### PR TITLE
Fix service_provider and start OpenStack Neutron Lbaas v2 agent

### PR DESCRIPTION
**Pull request by mjura**
This fix following error
```
  neutron.services.service_base [-] No providers specified for 'LOADBALANCERV2' service, exiting
```

To fully switch to LoadBalancerPluginv2 service, we have to change
`service_provider` option in `neutron_lbaas.conf` and start right
`openstack-neutron-lbaasv2-agent` service.

### CODE REVIEW

File: scripts/openstack-quickstart-demosetup [hunk 1]
```diff
@@ -462,8 +462,9 @@ do
 done
 
 if [ "x$with_tempest" = "xyes" ]; then
+    crudini --set /etc/neutron/neutron_lbaas.conf service_providers service_provider "LOADBALANCERV2:Haproxy:neutron_lbaas.services.loadbalancer.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default"
     crudini --set /etc/neutron/lbaas_agent.ini DEFAULT interface_driver neutron.agent.linux.interface.BridgeInterfaceDriver
-    start_and_enable_service openstack-neutron-lbaas-agent
+    start_and_enable_service openstack-neutron-lbaasv2-agent
```
Comment (aplanas) [inline scripts/openstack-quickstart-demosetup]: maybe  this needs to be merged / adjusted to this:

https://github.com/crowbar/crowbar-openstack/pull/513

Comment (mjura) [inline scripts/openstack-quickstart-demosetup]: As I see agent name stays `openstack-neutron-lbaasv2-agent`

Comment (mjura) [inline scripts/openstack-quickstart-demosetup]: Change `crowbar/crowbar-openstack#513` looks like not related

Comment (bmwiedemann) [inline scripts/openstack-quickstart-demosetup]: is that lbaasv2 available in Cloud5, too?

Comment (mjura) [inline scripts/openstack-quickstart-demosetup]: I assume that not, but this change will be backported only to `stable/mitaka` branch

Comment (aplanas) [inline scripts/openstack-quickstart-demosetup]: @mjura I was referring that maybe you need to set device_driver to "neutron_lbaas.drivers.haproxy.namespace_driver.HaproxyNSDriver"

Comment (mjura) [inline scripts/openstack-quickstart-demosetup]: @aplanas : For master branch and Mitaka option device_driver was deprecated already or at least it is configured right now with option `service_provider` which was causing this issue
